lib/resources/majors.tsx

```diff
@@ -2304,9 +2296,44 @@ const bsCS_Old: Major = {
       ],
     },
     {
-      name: 'Mathematics',
-      required_credits: 24,
+      name: 'Computer Science Classifications',
+      required_credits: 6,
       min_credits_per_course: 3,
+      description:
+        'At least one course in two different classification areas (Applications, Reasoning, Software, Systems) must be chosen in addition to Theory (Algorithms).',
+      criteria: 'CSCI-APPL[T]^OR^CSCI-SOFT[T]^OR^CSCI-SYST[T]^OR^CSCI-RSNG[T]',
+      pathing: 2,
+      fine_requirements: [
+        {
+          description: '<b>Application</b>',
+          required_credits: 3,
+          criteria: 'CSCI-APPL[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Software</b>',
+          required_credits: 3,
+          criteria: 'CSCI-SOFT[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Systems</b>',
+          required_credits: 3,
+          criteria: 'CSCI-SYST[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Reasoning</b>',
+          required_credits: 3,
+          criteria: 'CSCI-RSNG[T]',
+          double_count: ['N/A'],
+        },
+      ],
+    },
+    {
+      name: 'Mathematics',
+      required_credits: 12,
+      min_credits_per_course: 4,
```

Comment:
criteria could be more specific? Currently probstat satisfies `Mathematics` when it should not 
AS.110.108
AS.110.109
EN.553.171

Reply:
Addressed